src/quick-start.md

```diff
@@ -45,6 +45,7 @@ The following charts should be installed inside the `kubewarden` namespace in yo
 Kubewarden has two main components which you will interact with:
```

Comment:
```suggestion
Kubewarden has three main components which you will interact with:
```

Reply:
done in https://github.com/kubewarden/docs/pull/90/commits/3884c372cb0809d18ac9e102d03aef8387e41992